### PR TITLE
initial commit of one.com dns api

### PR DESCRIPTION
**Pull request by diseq**
initial support of one.com dns api

### CODE REVIEW

**Comment by Neilpang** [comment]
please check this guide first.
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by diseq** [comment]
ok. didn't pay attention to it. fixing according to guidline

File: dnsapi/dns_one.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  postdata="cmd=create"
+  postdata+="&subDomain=$mysubdomain"
+  postdata+="&priority="
+  postdata+="&ttl=600"
+  postdata+="&type=TXT"
+  postdata+="&value=$txtvalue"
+  postdata+="&csrft=$mycsrft"
+
+  response="$(_post "$postdata" "https://www.one.com/admin/dns-web-handler.do" "" "POST")"
+  echo $response
```
Comment (Neilpang) [inline dnsapi/dns_one.sh]: why `echo response`  here ?

**Comment by Neilpang** [comment]
how is going on ?

**Comment by diseq** [comment]
maybe i find time on the weekend to fix.

**Comment by diseq** [comment]
i cannot check for multiple domains. this script only allows the primarily registered one.

**Comment by diseq** [comment]
sorry. had a stupid mistake and domain check was missing

**Pull request closed without merging**